src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx

```diff
@@ -0,0 +1,182 @@
+     </td>
+     </tr>
+    <tr>
+    <td>
+        .NET agent 9.7.0
+    </td>
+    <td>
+        Microsoft Extensions Logging (MEL)
+    </td>
+    <td>
+        Default configuration (Log forwarding not enabled)
+    </td>
+    <td>
+        Not affected
+    </td>
+    </tr>
+    <tr>
+    <td>
+      .NET agent 9.8.0 through 10.0.0
+    </td>
+    <td>
+        Microsoft Extensions Logging (MEL)
+    </td>
+    <td>
+        Log forwarding disabled
+    </td>
+    <td>
+        Not affected
+    </td>
+    </tr>
+    <tr>
+    <td>
+     .NET agent 9.7.0 through 10.0.0
+    </td>
+    <td>
+      Microsoft Extensions Logging (MEL)
+    </td>
+    <td>
+        Configured to forward all Microsoft Extension Logging levels
+    </td>
+    <td>
+        Not affected
+    </td>
+     </tr>
+     <tr>
+    <td>
+        .NET agent before 9.7.0
+    </td>
+    <td>
+      n/a
+    </td>
+    <td>
+
+    </td>
+    <td>
+      Not affected
+    </td>
+     </tr>
+     <tr>
+    <td>
+    .NET agent 10.1.0 and later
+    </td>
+    <td>
+    Microsoft Extensions Logging (MEL)
+    </td>
+    <td>
+
+    </td>
+    <td>
+      Not affected
+    </td>
+     </tr>
+     <tr>
+    <td>
+        .NET agent all versions
+    </td>
+    <td>
+     n/a
+    </td>
+    <td>
+     Deployed in Linux
+    </td>
+    <td>
+      Not affected
+    </td>
+     </tr>
+    
+    </tbody>
+</table>
+
+### Fixed in: 
+*  New Relic .NET agent versions 10.1.0 and later
+
+### Recommended action: [#recommended]
+* Customers who use Microsoft Extension Logging should [upgrade to version 10.1.0 or later ](https://docs.newrelic.com/docs/apm/agents/net-agent/installation/update-net-agent/)
+* **Technical Links**: [Updating the .NET agent](https://docs.newrelic.com/docs/apm/agents/net-agent/installation/update-net-agent/)
+* **Workarounds**: Affected customers who cannot update their .NET agents to 10.1.0 or later at this time can [disable log forwarding](https://docs.newrelic.com/docs/logs/logs-context/disable-automatic-logging/). 
+
+## Technical details: [#technical-details]
+
+Version 10.1.0 remediates an error in the timing of the instrumentation point for Microsoft Extensions Logging to correctly send MEL data after the built-in log level filtering occurs. 
+
+## Timeline details [#timeline]
+
+This issue was introduced when New Relic added support for the Log Forwarding feature with Microsoft Extensions Logging (MEL) framework in .NET Core applications in .NET agent v 9.7.0 (April 4, 2022) and in .NET Framework applications in v 10.0.0 (July 19, 2022).  In version 9.7.0, the log forwarding feature was disabled by default, so customers using version 9.7.0.0 may only be affected by this issue if they have manually configured log forwarding.  
+
+In version 9.8.0 (May 5th, 2022), the Log Forwarding feature was enabled by default.  
+
+The issue was fixed with the release of .NET agent version 10.1.0, released on September 12, 2022. 
+
+## Frequently asked questions [#faq]
```

Comment:
You may need a line break here between the header and the first list element for this to render properly.